cOS/cOS.py

```diff
@@ -333,6 +333,34 @@ def getFrameRangeText(filename):
 	return filename + ' %d-%d' % \
 		(frameRange['min'], frameRange['max'])
 
+def getFileFromFrameRangeText(fileText):
+	'''
+	Supports 3 methods of import for imageSequences		 +	# Gets frame: 1001 of imageSequence
+ 	Uses cOS getFrameRange to find all images in matching sequence
+ 	Requires filename in format '../image.%04d.png' etc,
+ 	with %04d or other type of specification included in string
+ 	'''
+	filepath = normalizePath(fileText)
+	filePieces = filepath.split(' ')
+
+	# extracting filename from selection
+	if len(filePieces) == 2 and filePieces[1].split('-')[0].isnumeric():
+		filepath = filePieces[0].replace('%04d', filePieces[1].split('-')[0])
```

Comment:
do the same % regex from below
use format 

example:
frame = 6
image.%04d.exr
image.6.exr
image.0006.exr
filepath % frame